tests/test_resource.py

```diff
@@ -310,12 +324,13 @@ def setup_method(self, _test_method, _init_mock: MagicMock) -> None:
         self.resource = PaginatedCORSResource({}, [])
         self.resource.path = "/test/path"
         self.resource.urn = "/test/path"
-        self.resource.request = SimpleNamespace(
-            query_params={
-                "test": "param",
-                "another": "example",
+        req.query_params = MultiDict(
```

Comment:
It was being already changed that either way. Actually using SimpleNamespace was wrong from the beggining.

Reply:
`SimpleNamespace` is wrong when we talk about the type in general but if we need a simple object that contains the `query_params` attribute, it is a pretty nice simplification 😉 

I didn't see the wrong usage of `req` but generally, defining it as a global variable is a wrong idea, so it would be good to change that sooner than later (according to the TODO, instead of that, we should have a fixture) - can be done in the near future.